sdk/tables/tests.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       Timeout: 60
       MaxParallel: 12
       EnvVars:
-        AZURE_TEST_MODE: LIVE
+        AZURE_TEST_MODE: RECORD
```

Comment:
We run them in LIVE mode so that the results aren't persisted, it'll just run against the live service.

Reply:
I thought they would work mostly the same, but when I run in LIVE mode I get an error from azure core that `recordFile` is null (or something like that, I don't 100% precisely recall the error). I'm not sure why it works for RECORD but not for LIVE.

Reply:
Fixed